tests/regression/conftest.py

```diff
@@ -46,45 +44,51 @@ def pytest_addoption(parser: pytest.Parser) -> None:
     )
 
 
-@pytest.fixture(scope="module", autouse=True)
+@pytest.fixture(scope="session", autouse=True)
 def fxt_user_name(request: pytest.FixtureRequest) -> str:
     """User name to sign off the regression test execution.
 
     This should be given by the PyTest CLI option.
     """
-    user_name = request.config.getoption("--user-name")
+    user_name = request.config.getoption("--user-name", default=os.environ.get('USER_NAME'))
+    if user_name is None:
+        raise ValueError("Missing user name: Please provide it via the command-line option '--user-name' or set the 'USER_NAME' environment variable.")
```

Comment:
Why did you change `user_name` and the others as optional arguments from CLI? You should revert this changes and inject the variables needed in the tox commands.

Reply:
Done